tests/phpunit/tests/rest-api/rest-comments-controller.php

```diff
@@ -3386,8 +3478,73 @@ public function test_check_read_post_permission_with_invalid_post_type() {
 		$this->setExpectedIncorrectUsage( 'map_meta_cap' );
 
 		wp_set_current_user( self::$admin_id );
-		$request  = new WP_REST_Request( 'GET', '/wp/v2/comments/' . $comment_id );
+		$request  = new WP_REST_Request( $method, '/wp/v2/comments/' . $comment_id );
 		$response = rest_get_server()->dispatch( $request );
 		$this->assertSame( 403, $response->get_status() );
 	}
+
+	/**
+	 * @dataProvider data_readable_http_methods
+	 * @ticket 56481
+	 *
+	 * @param string $method HTTP method to use.
+	 */
+	public function test_get_items_only_fetches_ids_for_head_requests( $method ) {
```

Comment:
```suggestion
	public function test_get_items_returns_only_ids_for_head_requests( $method ) {
```
Suggestion per previous note.

Reply:
https://github.com/WordPress/wordpress-develop/pull/7970#discussion_r1876900479